.wordlist.txt

```diff
@@ -0,0 +1,17 @@
+# gpu_aware_mpi
```

Comment:
Can you move this to rocm-docs-core? Let's centralize the spell check as much as possible. We can use a local wordlist if for some reason the library will have nonstandard spelling.